lib/parser.js

```diff
@@ -42,19 +30,61 @@ function peg$subclass(child, parent) {
 }
 
 function peg$SyntaxError(message, expected, found, location) {
-  this.message = message;
-  this.expected = expected;
-  this.found = found;
-  this.location = location;
-  this.name = "SyntaxError";
-
-  if (typeof Error.captureStackTrace === "function") {
-    Error.captureStackTrace(this, peg$SyntaxError);
-  }
+  var self = Error.call(this, message);
+  if (Object.setPrototypeOf) {
+    Object.setPrototypeOf(self, peg$SyntaxError.prototype);
+  }
+  self.expected = expected;
+  self.found = found;
+  self.location = location;
+  self.name = "SyntaxError";
+  return self;
 }
 
 peg$subclass(peg$SyntaxError, Error);
 
+function peg$padEnd(str, targetLength, padString) {
+  padString = padString || " ";
+  if (str.length > targetLength) { return str; }
+  targetLength -= str.length;
+  padString += padString.repeat(targetLength);
+  return str + padString.slice(0, targetLength);
+}
+
+peg$SyntaxError.prototype.format = function(sources) {
+  var str = "Error: " + this.message;
+  if (this.location) {
+    var srcLines = [];
+    var src = null;
+    var k;
+    for (k = 0; k < sources.length; k++) {
+      if (sources[k].source === this.location.source) {
+        src = sources[k].text.split(/\r\n|\n|\r/g);
+        break;
+      }
+    }
+    var maxLine = this.location.start.line.toString().length;
+    if (!src) {
+      str += "\n at " + this.location.source + ":" + this.location.start.line + ":"
+          + this.location.start.column;
+    } else {
+      var line = src[this.location.start.line - 1];
+      str += "\n --> " + this.location.source + ":" + this.location.start.line + ":"
+          + this.location.start.column + "\n" + peg$padEnd("",maxLine)
+          + " |\n" + peg$padEnd(this.location.start.line.toString(), maxLine)
+          + " | " + line + "\n" + peg$padEnd("", maxLine) + " | "
+          + peg$padEnd("", this.location.start.column - 1);
+      if (this.location.start.line === this.location.end.line) {
+        str += peg$padEnd("", this.location.end.column - this.location.start.column, "^");
+      }
+      else {
+        str += peg$padEnd("", line.length - this.location.start.column + 1, "^");
```

Comment:
This is the line I can't reach.

Reply:
```
start = @&{
multiline
};
```
triggers that. If my fork would be used, I've changed that so it points only to `@`, therefore it will not work.